src/index.html

```diff
@@ -7,8 +7,58 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Inter&family=Poppins:wght@300;500&family=Roboto:wght@500&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="index.html">
+        <img src="images/logo.png" alt="some-logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav-ul">
+          <li class="li-nav"><a class="element is-active" href="#">Apple</a></li>
+          <li class="li-nav"><a class="element" href="#">Samsung</a></li>
+          <li class="li-nav"><a class="element" href="#">Smartphones</a></li>
+          <li class="li-nav">
+            <a
+               class="element element-x"
+               href="#"
+               data-qa="hover"
+            >Laptops & Computers
+            </a>
+          </li>
+          <li class="li-nav">
+            <a class="element" href="#">Gadgets
```

Comment:
```suggestion
            <a class="list__item--link" href="#">Gadgets
```
I would also try to be little bit more explicit, because every item on our webpage is `element`, so this kind of class name doesn't really mean anything, please try out the suggestion 😉